pyatlan/model/assets/__init__.py

```diff
@@ -25,6 +25,7 @@
 from .purpose import Purpose
 from .collection import Collection
 from .folder import Folder
+from .alteryx_workflow import AlteryxWorkflow
```

Comment:
This is a one-off customization (in-progress experiment) — we should remove from generated code.

Reply:
I have added code to prevent generation of attributes that have a type of a custom entity definition